javascript/node/selenium-webdriver/lib/webdriver.js

```diff
@@ -795,6 +798,15 @@ class WebDriver {
     if (typeof script === 'function') {
       script = 'return (' + script + ').apply(null, arguments);'
     }
+
+    if (script != undefined && script instanceof PinnedScript) {
```

Comment:
because of the falsy nature of JS we could probably change this to 
```suggestion
    if (script && script instanceof PinnedScript) {
```
JavaScript can conflate `null` and `undefined`... 😔

Reply:
Thank you!